src/screens/HomeScreens/components/TodoForm/TodoForm.js

```diff
@@ -1,7 +1,8 @@
-import React,{useState} from 'react'
+import React,{useState, useContext} from 'react'
 import {Form,Button,Row,Col} from 'react-bootstrap'
 import { useDispatch } from 'react-redux'
-import {addInfoAction} from '../../../../redux/actions'
+import { InfosContext } from '../../../../context/InfosContext'
+// import {addInfoAction} from '../../../../redux/actions'
 
 const TodoForm =() =>{
     const initalInfo = {
```

Comment:
can this `initialInfo` be moved to context?

Reply:
This one is just for the local state to store this form input, not for the global state. So I think it can't be moved to context. But I can add another one for the context